### ISSUE TITLE
0.20.0-1 "Oops! We couldn't load Uchiwa configuration." after upgrade

### ISSUE THREAD

**Issue by carljohnston1** [sensu/uchiwa#598]
After upgrading from 0.19.0-1 to 0.20.0-1 on (Oracle) EL7 I'm receiving "Oops! We couldn't load Uchiwa configuration." at the top of the right hand pane on every screen. 

My uchiwa configuration is:
```
{
  "uchiwa": {
    "user": "",
    "pass": "",
    "refresh": 5,
    "host": "0.0.0.0",
    "port": 3000
  },
  "sensu": [
    {
      "name": "Vagrant (vagrant-test)",
      "host": "sensu1",
      "port": 4567,
      "ssl": false,
      "timeout": 5
    }
  ]
}
```

![image](https://cloud.githubusercontent.com/assets/14154835/20371188/1a14ac38-acb7-11e6-8798-b0f31e5dba6f.png)

**Comment by palourde** [comment]
Hi,

Please make sure to restart the Uchiwa daemon and try to clear your browser cache.

If you still face this issue, could you provide us the logs in your console browser?

Thanks!

**Comment by carljohnston1** [comment]
Hi Simon,

Thanks for the quick follow-up; clearing the browser cache has removed the error message.

I'm not sure asking people to clear their caches is a good user experience though - can I provide you more information or assistance to get the upgrade seamless to the user?

Thanks,

Carl

**Comment by palourde** [comment]
I totally agree with you, this is why we just pushed a patch to prevent further issues when upgrading by disabling any cache of the index.html file: https://github.com/sensu/uchiwa/pull/597

It will be included in the 0.20.1 release.

Thanks!

**Comment by carljohnston1** [comment]
Awesome, thank you! 

And thanks for an awesome product too!
